platform/marvell-arm64/platform.conf

```diff
@@ -2,27 +2,70 @@
 
 # over ride default behaviour
 
+PLATFORM_AC5X="arm64-nokia_ixs7215_52xb-r0"
```

Comment:
this one is very similiar to this pr.

I am not sure if they are almost exactly the same.

[https://github.com/sonic-net/sonic-buildimage/pull/13500/files](https://github.com/sonic-net/sonic-buildimage/pull/13500/files)

I suggest to unified the arm64 installation file

Reply:
my comments on the other pr applied to this pr as well.

Reply:
@lguohan  I see 4 comments from you on the other PR related to platform.conf
1.	The comment about “boot_once” does not apply to our PR. We do not have that line
2.	Next comment about “bootcmd” behavior also does not apply to us. We do not do that.
3.	Merge code blocks in line 204-217 to the block 229-240 instead of writing 2 separate blocks
4.	comment about swap being added to sonic.

Are you referring to comments 3 and 4 being applied to this PR.

Reply:
Hi @lguohan 
We have addressed all your comments on the arm64 installation file from the other PR
As stated above I see 4 comments from you related to platform.conf file

Comment 1 and 2 does not apply to this PR as we do not have those lines of code.
Comment 3: The Code blocks have been consolidated into existing block
Comment 4: After careful analysis and testing, the arm64 on our A1 platform does not require swap so we have removed the blocks of code that created the swap partition 

Can you please review